overrides/groovy/postInit/Post-Initial/Main/Mod-Specific/ae2/misc.groovy

```diff
@@ -117,7 +117,7 @@ var applyProcessorChange = { OreDictIngredient lens, OreDictIngredient plate, It
 		.top(circuit)
 		.bottom(item('appliedenergistics2:material:20')) // Silicon
 		.output(processor)
-		.inscribe().register()
```

Comment:
Actually, one final change for clarity:

Can you add `.press` here?

Reply:
Unfortunately mobile GitHub doesn't let me edit code so can't do that till tomorrow unless u can just take a sec to add that in there.